src/input-mappings.js

```diff
@@ -1,47 +1,94 @@
-export default function registerInputMappings() {
-  AFRAME.registerInputMappings({
-    mappings: {
-      default: {
-        common: {
-          // @TODO these dpad events are emmited by an axis-dpad component. This should probalby move into either tracked-controller or input-mapping
-          dpadleftdown: "action_snap_rotate_left",
-          dpadrightdown: "action_snap_rotate_right",
-          dpadcenterdown: "action_teleport_down", // @TODO once once #30 lands in aframe-teleport controls this just maps to "action_teleport_aim"
-          dpadcenterup: "action_teleport_up", // @TODO once once #30 lands in aframe-teleport controls this just maps to "action_teleport_teleport"
-          touchpadpressedaxismovex: "translateX",
-          touchpadpressedaxismovey: "translateZ",
-          touchpadbuttonup: "stop_moving"
-        },
-        "vive-controls": {
-          menudown: "action_mute"
-        },
-        "oculus-touch-controls": {
-          xbuttondown: "action_mute",
-          gripdown: "middle_ring_pinky_down",
-          gripup: "middle_ring_pinky_up",
-          thumbsticktouchstart: "thumb_down",
-          thumbsticktouchend: "thumb_up",
-          triggerdown: "index_down",
-          triggerup: "index_up"
+const inGameActions = {
+  // Define action sets here.
+  // An action set separates "driving" controls from "menu" controls.
+  // Only one action set is active at a time.
+  default: {
+    move: { label: "Move" },
+    snap_rotate_left: { label: "Snap Rotate Left" },
+    snap_rotate_right: { label: "Snap Rotate Right" },
+    action_mute: { label: "Mute" },
+    action_teleport_down: { label: "Teleport Aim" },
+    action_teleport_up: { label: "Teleport" },
+    action_share_screen: { label: "Share Screen" }
+  }
+};
+
+const config = {
+  behaviours: {
+    default: {
+      "oculus-touch-controls": {
+        joystick: "oculus_touch_joystick_dpad4"
+      },
+      "vive-controls": {
+        trackpad: "vive_trackpad_dpad4"
+      }
+    }
+  },
+  mappings: {
```

Comment:
Most of the mappigns turn out pretty nice with the behaviors and activators, afim is coming along